### PR TITLE
Add 1.17 pipelines entry to the distro map

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): main only

Link to docs preview: `n/a`

QE review: `n/a`

Additional information: This PR enables building a new version of a standalone documentation set on docs.openshift.com. This PR does not alter any documentation content, so no documentation preview is available.

### CODE REVIEW

File: _distro_map.yml [hunk 1]
```diff
@@ -419,6 +419,9 @@ openshift-pipelines:
     pipelines-docs-1.16:
       name: '1.16'
       dir: pipelines/1.16
+    pipelines-docs-1.17:
+      name: '1.17'
+      dir: pipelines/1.17
```
Comment (max-cx) [inline _distro_map.yml:424]: @eromanova97, is the directory already created?

**Review by max-cx — APPROVED** [review]
LGTM

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Pull request merged**